### PR TITLE
Feature - Version bump to new upstream 2.2.0 release.

### PR DESCRIPTION
**Pull request by paraenggu**
* [Upstream release announcement](https://groups.google.com/d/msg/crc-mmbtools/V5T9RRyap5s/y40a5F3oAgAJ)
* [Upstream ChangeLog](https://github.com/Opendigitalradio/ODR-AudioEnc/blob/v2.2.0/ChangeLog)

### CODE REVIEW

File: odr-audioenc.spec [hunk 1]
```diff
@@ -51,7 +51,7 @@
 
 Name:           %{main_name}
 Version:        %{main_version}
-Release:        %{main_release}
+Release:        %{main_release}.1
```
Comment (hairmare) [inline odr-audioenc.spec]: Can we add this to the `%define main_release`?

File: odr-audioenc.spec [hunk 2]
```diff
@@ -93,7 +93,7 @@ file, a pipe, or to a ZeroMQ output compatible with ODR-DabMux.
 
 %package -n     %{toolame_dab_name}
 Version:        %{libtoolame_dab_version}
-Release:        %{libtoolame_dab_release}
+Release:        %{libtoolame_dab_release}.1
```
Comment (hairmare) [inline odr-audioenc.spec]: Same here, can we add this to `%define libtoolame_dab_release`?

File: odr-audioenc.spec [hunk 3]
```diff
@@ -105,7 +105,7 @@ in form of a library to be used with the encoder for the ODR-mmbTools
 
 %package -n     %{toolame_dab_name}-devel
 Version:        %{libtoolame_dab_version}
-Release:        %{libtoolame_dab_release}.1
+Release:        %{libtoolame_dab_release}.2
```
Comment (hairmare) [inline odr-audioenc.spec]: And another one that could be added to `%define libtoolame_dab_release`. This change also contradicts the release version of  `%{toolame_dab_name}-devel` vs. `%{toolame_dab_name}`.

**Review by hairmare — CHANGES_REQUESTED** [review]

File: odr-audioenc.spec (hunk 1 above)
Comment (paraenggu) [reply odr-audioenc.spec]: Yeah sure, `rpmdev-bumpspec` broke them and I didn't notice. Thanks for spotting it.

File: odr-audioenc.spec (hunk 2 above)
Comment (paraenggu) [reply odr-audioenc.spec]: Same as before - thanks.

File: odr-audioenc.spec (hunk 3 above)
Comment (paraenggu) [reply odr-audioenc.spec]: Properly incremented and fixed.

**Review by hairmare — APPROVED** [review]

**Pull request merged**